### ISSUE TITLE
This just crashes upon first request

### ISSUE THREAD

**Issue by Darker** [watson/http-echo-server#1]
I tried the server, I opened it in browser. Nothing happens, this pops up in console:

C:\Users\mareda\AppData\Roaming\npm\node_modules\http-echo-server\server.js:39
    throw err
    ^
```
Error: write ECONNRESET
    at exports._errnoException (util.js:1033:11)
    at Socket._writeGeneric (net.js:727:26)
    at Socket._write (net.js:746:8)
    at doWrite (_stream_writable.js:329:12)
    at writeOrBuffer (_stream_writable.js:315:5)
    at Socket.Writable.write (_stream_writable.js:241:11)
    at Socket.write (net.js:673:40)
    at Socket.<anonymous> (C:\Users\mareda\AppData\Roaming\npm\node_modules\http-echo-server\server.js:18:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
```

**Comment by jakeprem** [comment]
What version of Node are you running? I don't remember if it was that exact error but I've seen it crash on Node 6.

Node 8.10 and 8.11 have both worked fine for me.

**Comment by watson** [comment]
This have been fixed in v2.0.0
